src/components/Common/CustomButton/CustomButton.jsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import classes from './CustomButton.module.css';
+const CustomButton = (props) => {
+    return (
+        <div>
+            <button
+                style={{
+                    color: props.styles.color,
+                    backgroundColor: props.styles.backgroundColor,
+                    height: props.styles.height || "32px",
+                    width: props.styles.width || "100",
```

Comment:
change "100" to "100%". If it is taking 100% even if passing "100" then keep as it is. 
Otherwise change to "100%"

Reply:
how to differentiate between 100 and 100%